test/cpp/end2end/hybrid_end2end_test.cc

```diff
@@ -207,6 +207,9 @@ class HybridEnd2endTest : public ::testing::Test {
     ServerBuilder builder;
     builder.AddListeningPort(server_address_.str(),
                              grpc::InsecureServerCredentials());
+    // Always add a sync unimplemented service: we rely on having at least one
+    // synchronous method to get a listening cq
+    builder.RegisterService(&unimplemented_service_);
```

Comment:
We've at least one test in the suite that overrides all methods as async or generic, meaning there's no sync part - meaning there's no sync listener (and that test needs one).

We could either split out some code for that specific test (which one is escaping me at the minute), or live with this to ensure that there's a sync listener around.